guides/user-manual/api_auth.rst

```diff
@@ -17,33 +17,53 @@ There a few options for authenticating against the M&M API:
 
 .. _api-login-headers:
 
 Authorization Headers
 ---------------------
 
-With the introduction of Men&Mice Web Services and the REST API in version 7.3, it's now also possible to authenticate via HTTP headers, by adding a Basic Authorization header, or with a Negotiate Authorization header, if the client supports it.
+It's possible to authenticate via HTTP headers by adding a Basic Authorization header or a Negotiate Authorization header, if supported bye the client.
 
 .. note::
-  If AuthorizationMethods is not defined in the preference file mmWS will only accept basic authorization.
+  If AuthorizationMethods is not defined in the preference file, mmWS will only accept Basic Authorization.
 
-By using authorization headers for authentication, the Login command becomes unnecessary, and the session ID is not used. The "session" parameter for all the commands must be omitted, and not left blank.
+Using authorization headers for authentication eliminates the need for the Login command, which makes the session ID unnecessary. The "session" parameter should be omitted entirely from all commands, not just left blank.
 
-With this authentication method, multiple M&M Web Service instances can be deployed and used simultaneously behind a load balancer to provide a redundancy that is transparent to the API client.
+This method of authentication allows multiple Micetro Web Interfaces to be deployed behind a load balancer, providing redundancy that is seamless to the API client.
+
+However, session IDs are only valid at the API endpoint where they were generated. Therefore, if an API client using a session ID is redirected to different Web Interface/API Endpoint, it will receive an error indicating that the session ID is invalid or has expired. The client must then log in again to obtain a new session ID before proceeding.
+
+The authorization header authentication method is only available for JSON-RPC and REST APIs, and only if using the Micetro Web Interface as an endpoint (URL is <server>/mmws/api or <server>/mmws/json).
+
+
+Bearer Authentication
+^^^^^^^^^^^^^^^^^^^^^
+Bearer authentication, also known as token-based authentication, is a common method for securing Web APIs and is the preferred method for interacting with and authenticating the Micetro APIs.
+
+A bearer token is an encrypted string typically generated by the server following a login request. The token serves as proof of authorization. When making requests to the Micetro API, send this token in the Authorization header:
+
+.. code-block::
+
+  Authorization: Bearer <token>
+
+The Micetro API checks the token's validity. If it is valid, the client is granted access to the requested resource.
+
+Ensure that your Micetro endpoints use HTTPS to prevent interception of usernames and passwords during communication.
```

Comment:
perhaps this should follow the instructions how to generate the session, in a warning bubble, etc?